### ISSUE TITLE
Sometimes tapping a notification leads to blank screen

### ISSUE THREAD

**Issue by tonyr59h** [wordpress-mobile/WordPress-Android#2423]
Would be great to get to the bottom of the missing notification, but if it can't be found the previous activity should be loaded.

**Comment by rachelmcr** [comment]
I looked and can't find any recent reports of missing/blank notifications, and I can't find a way to reproduce the issue.

I agree that loading the previous activity would be good fallback behavior, but seems low priority unless we receive more reports about blank notifications.

**Comment by mzorz** [comment]
I think this has been already taken care of by this commit https://github.com/wordpress-mobile/WordPress-Android/commit/c1d8950ca73e4db116c9e6e107490a86d8f195e0#diff-876fbccc3a7b4b70ff6167a96b5c0f1c

I have tested and confirmedthe following scenario:

1) create a new comment from a 3rd user, thus triggering the PNs on my logged in Android device
2) deleting the comment completely (first trashing, then deleting the trash) on a third device (i.e. iOS)
3) tap on the received PN in step 1 on the Android device

The Toast should be triggered, and the activity should be finished, without leaving the screen blank - and the Notifications screen is shown.

Closing this one.
